systems/registry/site/pkg/config.go

```diff
@@ -35,7 +35,7 @@ type Config struct {
 }
 
 type HttpServices struct {
-	InventoryClient string `defaut:"localhost:8080"`
+	InventoryClient string `defaut:"http://api-gateway-inventory:8080"`
```

Comment:
Did you test this with `HttpServices` change ?

Reply:
yes